zegami_sdk/collection.py

```diff
@@ -260,14 +260,17 @@ def get_image_urls(self, rows, source=0, generate_signed_urls=False,
         By default the uploaded images are fetched, but it's possible to fech e.g. the thumbnails
         only, by providing an alternative imageset id.
         """
-        # Turn the provided 'rows' into a list of ints
+        # Turn the provided 'rows' into a list of ints.
+        # If 'rows' are not defined, get all rows of collection.
         if type(rows) == pd.DataFrame:
```

Comment:
Small suggestion for restructuring logic:

```suggestion
        if not rows:
             < empty row logic >
        else:
            if type(rows) == pd.DataFrame:
                ...
            elif ...
            else:
                raise ValueError('Invalid rows argument, \'{}\' not supported'.format(type(rows)))            
```